File: lemur-app/src/main/java/io/github/malczuuu/lemur/app/core/player/PlayerDetails.java
package io.github.malczuuu.lemur.app.core.player;

import io.github.malczuuu.lemur.app.domain.player.PlayerStatus;
import java.time.Instant;
import org.jspecify.annotations.Nullable;

public record PlayerDetails(
    String id,
    String name,
    int rating,
    PlayerStatus status,
    @Nullable Instant createdDate,
    @Nullable Instant lastModifiedDate,
    Long version) {}

File: lemur-app/src/main/kotlin/io/github/malczuuu/lemur/app/core/player/PlayerDetails.kt
package io.github.malczuuu.lemur.app.core.player

import io.github.malczuuu.lemur.app.domain.player.PlayerStatus
import kotlin.time.Instant

data class PlayerDetails(
    val id: String,
    val name: String,
    val rating: Int,
    val status: PlayerStatus,
    val createdDate: Instant?,
    val lastModifiedDate: Instant?,
    val version: Long,
)
